source/features/action-used-by-link.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import * as pageDetect from '../libs/page-detect';
+import searchIcon from 'octicon/search.svg';
+
+function init(): void {
+	const actionRepo = (select('.py-3.border-bottom.border-gray-light .octicon.octicon-issue-opened.text-gray-dark.mr-2')!.parentElement as HTMLAnchorElement)
```

Comment:
JMHO
```suggestion
	const actionRepo = (select('.octicon-issue-opened.text-gray-dark')!.parentElement as HTMLAnchorElement)
		.pathname.slice(1)
		.replace('/issues', '');
```
Its rare that github styles their own icons (from my experience)